config.yml

```diff
@@ -5,6 +5,8 @@ defaults:
     exchangePrefix: v1/
```

Comment:
This (`exchangePrefix`) is never used and confused the heck out of me for a while..